docs/specs/cross-chain-htlc.md

```diff
@@ -0,0 +1,80 @@
+# Payment Channel Cross-Chain Protocol with HTLC
+
+## Synopsis 
+
+In the rapidly evolving world of blockchain technology, interoperability between different networks is becoming increasingly crucial. One of the most promising solutions to this challenge is the use of payment channel cross-chain protocols based on Hash Time-Locked Contracts (HTLCs). This post will delve into how HTLCs can be used to ensure atomic payments across different blockchain networks using the same preimage.
+
+## What is HTLC?
+
+Hash Time-Locked Contracts (HTLCs) are a type of smart contract used to facilitate conditional payments. They are designed to ensure that a transaction is either completed within a specified timeframe or canceled. The key components of an HTLC are:
+
+1. **Hashlock**: A cryptographic hash of a secret (preimage) that must be revealed to complete the transaction.
+2. **Timelock**: A time constraint that ensures the transaction is either completed within a certain period or reverted.
+
+## Cross-Chain Payments with HTLC
+
+Cross-chain payments involve transferring value across different blockchain networks, which can be difficult due to the absence of direct interoperability. To address this challenge, Hashed Time Lock Contracts (HTLCs) offer a solution by facilitating atomic swaps. These swaps guarantee that transactions are either executed in full or not at all, eliminating the possibility of partial transfers.
+
+To ensure stability in cross-chain payments, the assets utilized in different blockchain networks must maintain a fixed swap ratio. For instance, the Bitcoin payment channel exclusively supports BTC. Meanwhile, CKB can incorporate a wrapped BTC token via the UDT channel, establishing a consistent 1:1 ratio with Bitcoin.
```

Comment:
I reality, the swap ratio will change with time, how can we keep it update to date?

Reply:
Assume that wrapped BTC is always 1:1 with BTC, or use an external script to get the ratio when proposing the exchange order.